extract_assets.py

```diff
@@ -16,13 +17,13 @@ def ExtractFile(xmlPath, outputPath, outputSourcePath):
         # Don't extract if another file wasn't extracted properly.
         return
 
-    execStr = "tools/ZAPD/ZAPD.out e -eh -i %s -b baserom/ -o %s -osf %s -gsf 1 -rconf tools/ZAPDConfigs/MqDbg/Config.xml" % (xmlPath, outputPath, outputSourcePath)
-    
+    execStr = f"tools/ZAPD/ZAPD.out e -eh -i {xmlPath} -b baserom/ -o {outputPath} -osf {outputSourcePath} -gsf 1 -rconf tools/ZAPDConfigs/MqDbg/Config.xml {ZAPDArgs}"
```

Comment:
I thought we were avoiding f-strings for mac compatibility (some macs do be having very old python)

Reply:
I think that all macs have python 2.7 by default. But to build the repo one would need to get python 3.X anyways

Reply:
And, this script uses python3 anyway.

Reply:
f-strings were added in python 3.6, I don't know anything about macs but it seems like some people with macs often have very old python versions
I'm fine to keep them, I don't plan on using outdated macs myself :p

Reply:
ZAPD doesn't build on a Mac with pre C++17 support anyway.